src/types.rs

```diff
@@ -15,6 +17,12 @@ pub struct Coordinate<T>
     pub y: T,
 }
 
+impl<T: Float> From<(T, T)> for Coordinate<T> {
```

Comment:
I was just trying to mimic the `Point::new` constructor, which is (x, y). I'm not 100% wedded to any one way.

Perhaps I am unsure about a fundamental point of this crate. I thought it represented geometric objects in any coordinate reference system, and hence the X/Y order makes sense, since another CRS/SRS would not be lat/lon. However if everything is aimed to be lat/lon, that changes things.

I am willing to split this part off (perhaps into another PR).